src/controllers/playback/video/index.html

```diff
@@ -33,7 +33,7 @@ <h3 class="osdTitle"></h3>
                         <span class="xlargePaperIconButton material-icons fiber_manual_record" aria-hidden="true"></span>
                     </button>
 
-                    <button is="paper-icon-button-light" class="btnPreviousTrack autoSize hide" title="${PreviousTrack}">
+                    <button is="paper-icon-button-light" class="btnPreviousTrack autoSize hide" title="${PreviousTrack} (P)">
```

Comment:
Other bindings are in lower case.
Also, we need to add `aria-label` because now `title` contains shortcut.

Reply:
These bindings require the shift key. Will add aria-labels's.

Reply:
> These bindings require the shift key.

Then it should include `Shift+`. Maybe `Shift+P` and change the rest to upper case?

Reply:
https://github.com/jellyfin/jellyfin-web/pull/5759/commits/ff1157ab9dc9cffa0bcea81ca27d842efa3c18e1

Reply:
Can't find 'Play (k)' to make the change there as well. 🤔

Reply:
https://github.com/jellyfin/jellyfin-web/pull/5759/commits/b681184e9903d0bfd4ece4a9ff5b88df44c1ba56